tests/test_binning.py

```diff
@@ -17,17 +17,35 @@
 from sed.binning.utils import bin_edges_to_bin_centers
 from .helpers import get_linear_bin_edges
 
-sample = np.random.randn(int(1e2), 3)
+sample = np.random.randn(int(1e5), 3)
 columns = ["x", "y", "z"]
 sample_df = pd.DataFrame(sample, columns=columns)
-bins: Sequence[int] = tuple(np.random.randint(5, 50, size=3))
+bins: Sequence[int] = tuple(np.random.randint(5, 50, size=3, dtype=int))
 ranges_array = 0.5 + np.random.rand(6).reshape(3, 2)
 ranges_array[:, 0] = -ranges_array[:, 0]
 ranges: Sequence[tuple] = tuple(tuple(r) for r in ranges_array)
 
-arrays = [
-    get_linear_bin_edges(np.linspace(r[0], r[1], b))
-    for r, b in zip(ranges, bins)
+arrays = [get_linear_bin_edges(b, r) for r, b in zip(ranges, bins)]
+
+sample_int = np.random.randint(
+    low=60000,
+    high=70001,
+    size=(int(1e5), 3),
+    dtype=int,
+)
+bins_int = tuple(np.random.randint(10, 300, size=3, dtype=int))
+ranges_int = [
+    tuple(np.sort(np.random.randint(60000, 70000, size=2, dtype=int))),
```

Comment:
why repeating the same tuple 3 times?

Reply:
I understand the code. My point was about repeating three times the same line. Why not something like:
```
ranges_int = [tuple(np.sort(np.random.randint(60000, 70000, size=2, dtype=int))) for _ in range(3)]
```

Reply:
I see, this is probably the pythonic way of writing this. I personally find it a bit more difficult to understand, but I can change it.